packages/flutter_test/lib/src/matchers.dart

```diff
@@ -354,6 +354,57 @@ Matcher coversSameAreaAs(Path expectedPath, { required Rect areaToCompare, int s
 /// ```
 /// {@end-tool}
 ///
+/// Fonts can be rendered differently according with the platform or the Flutter
+/// version in which the test is run. While the difference may be minimal, it's
+/// still valuable to make the test fail. For example, a golden file generated
+/// on Windows is going to differ from the one produced by another operating
+/// system. Even on the same platform, if the generated golden is tested with a
+/// different Flutter version, the test may fail.
+///
+/// By default, the Flutter framework uses a font called 'Ahem' which shows
+/// squares instead of characters. To make golden tests more valuable, there
+/// is the possibility to render images using other more human-readable fonts.
+/// For example, this is how to load the 'Roboto' font for a golden test:
```

Comment:
The statement about testing fonts being more valuable is arguable. 🙃 

```suggestion
/// By default, the Flutter framework uses a font called 'Ahem' which shows
/// squares instead of characters. It is possible to render images using
/// custom fonts. For example, this is how to load the 'Roboto' font for a
/// golden test:
```